app/javascript/dashboard/components/CustomAttribute.vue

```diff
@@ -247,17 +245,12 @@ export default {
       }
     },
     onUpdate() {
-      const updatedValue =
-        this.attributeType === 'date'
-          ? format(new Date(this.editedValue), DATE_FORMAT)
-          : this.editedValue;
-
```

Comment:
Removing the formatting as it subtracts 1 or 2 days from the date in different time zones.